README.md

```diff
@@ -26,20 +26,20 @@ TiKV uses Raft to support Geo-Replication. We have ported etcd's raft implementa
 This project requires rust nightly, otherwise it can't build.
 
 ```
-# 1. Install Rust Nightly
+# 1. Install rust nightly.
 curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
 
-# 2. Install RocksDB Library
-
-# 2.1 [optional step] Upgrade Your GCC To Version At Least 4.7 To Get C++11 Support
+# 2. Install RocksDB
+# Please view RocksDB's offical install document at https://github.com/facebook/rocksdb/blob/master/INSTALL.md .
 
+# 2.1 [optional step] Upgrade your GCC to version at least 4.7 to get C++11 support
```

Comment:
Since RocksDB already have the install manual, we should remove section 2.1. And merge section 2.2 to section 2.